src/intents.js

```diff
@@ -46,6 +46,14 @@ function injectService (url, element, intent, data) {
         return event.source.postMessage(data, event.origin)
       }
 
+      if (handshaken && event.data.type === `intent-${intent._id}:size`) {
+        ['width', 'height', 'maxWidth', 'maxHeight', 'minWidth', 'minHeight'].forEach(prop => {
```

Comment:
@CPatchane I actually use maxHeight and maxWidth in https://github.com/doubleface/cozy-client-js/blob/389578a203c7152e9e239b55db2a6e3843c852ad/src/intents.js#L158 since it allows the modale to adapt it's size if the resolution lower than expected.